src/ray/raylet/local_object_manager.cc

```diff
@@ -89,7 +89,8 @@ void LocalObjectManager::FlushFreeObjectsIfNeeded(int64_t now_ms) {
   }
 }
 
-int64_t LocalObjectManager::SpillObjectsOfSize(int64_t num_bytes_required) {
+int64_t LocalObjectManager::SpillObjectsOfSize(int64_t num_bytes_required,
```

Comment:
Can you rename this to `num_bytes_to_spill` to match the other variable name? I found it a bit confusing because it sounds like we're required to spill `num_bytes_required` too.